packages/template-block/src/components/TemplateText.tsx

```diff
@@ -93,17 +94,30 @@ export const TemplateText = ({
         [appBridge, blockId, description, isEditing, saveDescription, templateTextKey],
     );
 
+    const titleClasses = useMemo(() => {
```

Comment:
Wouldn't use `useMemo` as its not a heavy operation nor does the `prop` often change